src/components/Executions/ExecutionDetails/NodeExecutionDetailsPanelContent.tsx

```diff
@@ -218,13 +238,62 @@ const NodeExecutionTabs: React.FC<{
     );
 };
 
+const WorkflowTabs: React.FC<{
+    workflow: Workflow;
+    nodeId: string;
+}> = ({ workflow, nodeId }) => {
+    const styles = useStyles();
+    const tabState = useTabState(tabIds, tabIds.inputs);
+    const commonStyles = useCommonStyles();
+    let tabContent: JSX.Element | null = null;
+    const currentNode = workflow.closure?.compiledWorkflow.primary.template.nodes.find(
```

Comment:
This will only work for top-level nodes; you need to also collect nested nodes (Subworkflows, branches, dynamic, etc).